src/components/org/OrgChart/OrgChart.stories.tsx

```diff
@@ -179,20 +179,39 @@ const position: Position = {
 };
 const cardStyle = {
```

Comment:
Is cardStyle used, or could we get rid of it?

Reply:
Removed